openapi/multi-yaml/components/schemas/DisputeResponse.yaml

```diff
@@ -0,0 +1,53 @@
+type: object
+properties:
+  additional_statement:
+    type: string
+    description: any additional evidence or statements
+  cancelation_policy_disclosure:
+    type: string
+    description: an explanation of how and when the customer was shown your refund policy prior to purchase
```

Comment:
Should this refer to cancellation instead of refund?

Reply:
It's the same as in stripe
https://docs.stripe.com/api/disputes/object#dispute_object-evidence-cancellation_policy_disclosure

Reply:
Interesting. I'd suspect that is a mistake in the stripe docs because we do have a separate `refund_policy_disclosure`. 
I would still change this to cancelation policy but happy to get other opinions.